src/test/java/com/sap/cloudfoundry/client/facade/ITVariable.java

```diff
@@ -0,0 +1,51 @@
+package com.sap.cloudfoundry.client.facade;
+
+import java.util.Set;
+
+public class ITVariable {
+
+    private static final String CF_API_ENV = "CF_API";
+    private static final String USER_EMAIL_ENV = "CF_USER_EMAIL";
+    private static final String USER_PASSWORD_ENV = "CF_USER_PASSWORD";
+    private static final String ORG_ENV = "CF_ORG";
+    private static final String SPACE_ENV = "CF_SPACE";
+    private static final String DOMAIN_NAME_ENV = "DOMAIN_NAME";
+
+    private static final String CF_API_PROPERTY = "cf.api";
+    private static final String USER_EMAIL_PROPERTY = "user.email";
+    private static final String USER_PASSWORD_PROPERTY = "user.password";
+    private static final String ORG_PROPERTY = "org";
```

Comment:
actually I wanted to refrain from prefix "cf." for system properties because they will be used only for the running tests.
On the other side env variables are applied for all processes, and one variable "USER" will be misleading in the env